tools/AndroidDependencies/ProjectSettings/AudioManager.asset

```diff
@@ -0,0 +1,17 @@
+%YAML 1.1
```

Comment:
unity n00b question. Do we need all of these assets to use a few line script?

Reply:
unity creates them by default

Reply:
It's the minimum requirements for something to be considered a Unity project, AFAIK. Maybe @firtina-improbable has more context on that? I can try removing the assets to check if it will still work, though I did some experiments with removing things.

Reply:
For a unity project, these files are good to add to source control.

But, why do we need a new unity project for this? Can't we add them to the main repo instead?

Reply:
They are CI specific.

Reply:
Should we create more generic project for CI? Or is it better to keep it tied to Android for now?

Reply:
even if they are CI specific, it adds too much overhead vs. a script within other repo that only gets used in CI

Reply:
or, can add it to the test project I guess?

Reply:
I know it's not ideal, but considering it's CI only I think this is a nice separation of concerns. It wouldn't make sense to me to have it in the Test Project or the GDK project as it's not related to either of these. It's literally setting up Unity preferences. If people strongly believe it shouldn't be here, I can move it but I think this is the most logical place, albeit the unnecessary overhead.

Reply:
I agree this doesn't belong in the test project. Is there no other way of changing Unity preferences? If not, we should definitely raise a post/issue as this is a wacky workflow. 

Also we aren't the only ones who have to build mobile clients in CI - so I'm surprised that this is how its done

Reply:
I see, imo that's OK